lib/DXIL/DxilModule.cpp

```diff
@@ -1826,6 +1826,55 @@ bool DxilModule::StripReflection() {
   return bChanged;
 }
 
+bool DxilModule::StripNamesSensitiveToDebug() {
+  bool bChanged = false;
```

Comment:
I know this file uses Hungarian notation for variable naming, but that isn't consistent with LLVM style guidelines which we want all new code to follow (see: [DXC's Contributing](https://github.com/microsoft/DirectXShaderCompiler/blob/main/CONTRIBUTING.md#coding-guidelines) & [LLVM Coding Style: Names](https://github.com/microsoft/DirectXShaderCompiler/blob/main/docs/CodingStandards.rst#name-types-functions-variables-and-enumerators-properly)).